styles/variables.css

```diff
@@ -0,0 +1,32 @@
+:root {
+  /* ----- typography ----- */
+  --h1-font-size: 4rem;
+  --h2-font-size: 3rem;
+  --h3-font-size: 2.25rem;
+  --h4-font-size: 1.5rem;
+  --paragraph-font-size: 1.25rem;
+
+  /* ----- line-hight ----- */
+  --h1-line-height: 4.75rem;
+  --h2-line-height: 3.6rem;
+  --h3-line-height: 2.7rem;
+  --h4-line-height: 1.75rem;
+  --paragraph-line-height: 1.5rem;
+
+  /* ----- colors ----- */
+  --color-orange: #F6A721;
+  --color-blue: #03628A;
+  --color-blue-light: #6DAEC7;
+  --color-white: #FFF;
+  --color-grey: #D0D0D0;
+  --color-black: #282828;
+
+  /* ----- spacing ----- */
+  --space-xs: 4px;
```

Comment:
we may want to translate some of them to rems based use case - sometimes some padding/margin may need to scale respectively so we could use smth like https://dev.to/nikolab/convert-px-to-rem-using-sass-3-methods-4ep2

Reply:
@roertbb you mean changing only pixels to rems manualy or create that function for every 'length' units?